<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FourierTransform</class>
 <widget class="QWidget" name="FourierTransform">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>805</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" rowstretch="9,1">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gLayFFT"/>
   </item>
   <item row="0" column="1">
    <layout class="QGridLayout" name="gLayIFFT"/>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="vLaySampleRate">
       <item>
        <widget class="QLabel" name="labelSampleRate">
         <property name="text">
          <string>sample rate</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="hLaySampleRate">
         <item>
          <widget class="QSpinBox" name="spinBoxSampleRate">
           <property name="maximum">
            <number>999999</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelHz">
           <property name="text">
            <string>Hz</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="vLayRange">
       <item>
        <widget class="QToolButton" name="tBtnRange">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Range</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="hLayRange">
         <item>
          <widget class="QDoubleSpinBox" name="doubleSpinBoxStart">
           <property name="maximum">
            <double>999999.989999999990687</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_">
           <property name="text">
            <string>-</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="doubleSpinBoxEnd">
           <property name="maximum">
            <double>999999.989999999990687</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
